src/coupling/hc_explicit_sequential.cc

```diff
@@ -117,14 +113,15 @@ HC_ExplicitSequential::HC_ExplicitSequential(Input::Record in_record)
     // TODO: optionally setup transport objects
     Iterator<AbstractRecord> it = in_record.find<AbstractRecord>("secondary_equation");
     if (it) {
-    	transport_reaction = (*it).factory< AdvectionProcessBase, Mesh &, const Input::Record>(*mesh, *it);
+    	secondary_eq = (*it).factory< AdvectionProcessBase, Mesh &, const Input::Record >(*mesh, *it);
```

Comment:
In my opinion, it would take little effort to enable THC simulation with coupling H->C and H->T. But I would postpone it to a separate issue.